dev/com.ibm.ws.concurrent.mp_fat/test-applications/MPConcurrentApp/src/concurrent/mp/fat/web/MPConcurrentTestServlet.java

```diff
@@ -5567,6 +5573,7 @@ public void testWithContextCapture_CompletionStage_builder() throws Exception {
      * CompletionStage that is backed by a ThreadContext rather than a managed executor.
      */
     @Test
+    @SkipForRepeat(MPContextProp11RepeatAction.ID)
```

Comment:
@njr-11 This test fails with version 1.1, when you review please decide if a follow up change is needed to fix that.

Reply:
Also expected, per previous comments.